desktop/core/src/desktop/lib/botserver/data/queries.json

```diff
@@ -0,0 +1,14 @@
+[
+  {
+    "name": "Customer Ticket Analysis",
+    "query": "SELECT 1"
+  },
+  {
+    "name": "Employee Sales",
+    "query": "SELECT 2"
+  },
+  {
+    "name": "Quaterly Revenue Query",
+    "query": "SELECT 3"
```

Comment:
Put queries to sample Hue sample tables? (sample_07, weblogs) With `${variables}` too?

Reply:
PS: we can probably move this out of into a `assistant` in https://github.com/cloudera/hue/tree/master/desktop/libs/metadata/src/metadata as this is can be re-used within the Editor itself, not just Slack.